File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.js
function solution(num_list) {
    var answer = 0;
    let multi = num_list.reduce((a, b) => a * b, 1)
    let sum = num_list.reduce((a, b) => a + b, 0)
    answer = multi > Math.pow(sum, 2) ? 0 : 1
    return answer;
}

File: 프로그래머스/0/181929. 원소들의 곱과 합/원소들의 곱과 합.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        int multi = 1;
        int sum = 0;
        for(int a : num_list){
            multi = multi * a;
            sum += a;
        }
        answer = (multi < sum * sum) ? 1 : 0; 
        return answer;
    }
}